components/uid-textarea/package.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "uid-textarea",
+  "description": "An input with label",
+  "homepage": "https://github.com/bonitasoft/web-components/components/uid-textarea#readme",
+  "license": "GPLv2",
+  "version": "1.0.1",
+  "main": "dist/src/index.js",
+  "module": "dist/src/index.js",
+  "files": [
+    "assets",
+    "dist/uid-textarea.bundle.min.js",
+    "src/UidTextarea.ts",
+    "src/uid-textarea.ts",
+    "test"
+  ],
+  "exports": {
+    ".": "./dist/src/index.js",
+    "./uid-textarea.js": "./dist/src/uid-textarea.js"
+  },
+  "scripts": {
+	"gen-widget": "cwb gen-widget --propertiesFile uidTextarea.json --webComponentBundle lib/uid-textarea.es5.min.js --outputDir uid_widget",
```

Comment:
Remove this (duplicated), it has been added by widget duplicate. It has been fixed now, please upate your sdk.